modules/navier_stokes/test/tests/postprocessors/pressure_drop/tests

```diff
@@ -0,0 +1,134 @@
+[Tests]
+  design = 'PressureDrop.md'
+  issues = '#23685'
+  [fe]
+    type = CSVDiff
+    input = drop_insad.i
+    csvdiff = straight_insad.csv
+    abs_zero = 1e-8
+    requirement = 'The system shall be able to compute the pressure drop in a straight channel with a finite element incompressible Navier Stokes model.'
+    cli_args = "Mesh/inactive='diverging_mesh' Outputs/file_base='straight_insad'"
+    ad_indexing_type = 'global'
+  []
+  [fe_diverging]
+    requirement = 'The system shall be able to compute the pressure drop in a diverging channel with a finite element incompressible Navier Stokes model'
+    [regular]
+      type = CSVDiff
+      input = drop_insad.i
+      csvdiff =  diverging_insad.csv
+      abs_zero = 1e-8
+      detail = 'with a regular face pressure evaluation'
+      cli_args = "Outputs/file_base='diverging_insad'"
+      ad_indexing_type = 'global'
+    []
+    [weighted]
+      type = CSVDiff
+      input = drop_insad.i
+      csvdiff =  diverging_insad_weighted.csv
+      abs_zero = 1e-8
+      cli_args = "Outputs/file_base='diverging_insad_weighted'
+                  Postprocessors/pdrop_total/weighting_functor='velocity' Postprocessors/pdrop_mid1/weighting_functor='velocity'
+                  Postprocessors/pdrop_mid2/weighting_functor='velocity' Postprocessors/pdrop_mid3/weighting_functor='velocity'"
+      detail = 'with a pressure drop face evaluation weighted by the local velocity'
+      ad_indexing_type = 'global'
+    []
+  []
+
+  [insfv_straight]
+    requirement = 'The system shall be able to compute the pressure drop in a frictionless heated straight channel with a finite volume incompressible Navier Stokes model.'
+    [regular]
+      type = CSVDiff
+      input = drop_insfv.i
+      csvdiff =  straight_insfv.csv
+      abs_zero = 1e-8
+      cli_args = "Mesh/inactive='diverging_mesh' Outputs/file_base='straight_insfv' -pc_type lu -pc_factor_shift_type NONZERO"
+      detail = 'with a regular face pressure evaluation'
+      ad_indexing_type = 'global'
+    []
+    [weighted]
+      type = CSVDiff
+      input = drop_insfv.i
+      csvdiff =  straight_insfv_weighted.csv
+      abs_zero = 1e-8
+      cli_args = "Mesh/inactive='diverging_mesh' Outputs/file_base='straight_insfv_weighted'
+                  -pc_type lu -pc_factor_shift_type NONZERO
+                  Postprocessors/pdrop_total/weighting_functor='velocity' Postprocessors/pdrop_mid1/weighting_functor='velocity'
+                  Postprocessors/pdrop_mid2/weighting_functor='velocity' Postprocessors/pdrop_mid3/weighting_functor='velocity'"
+      detail = 'with a pressure drop face evaluation weighted by the local velocity'
```

Comment:
```suggestion
      detail = 'with a pressure drop face evaluation weighted by the local velocity.'
```

Reply:
@cticenhour should there be a conjunction on the last detail? (e.g. and)

Reply:
APA style applies `, and` at the end of the next-to-last detail to make the itemized list a complete sentence. I have been following that approach.